src/App.css

```diff
@@ -40,3 +40,43 @@
   padding: 0.5rem 1rem;
   color: rgba(0,0,0,.5);
 }
+
+
+/* Moet worden toegevoegd om material icons werkend te maken. */
```

Comment:
what is this language 😭

Reply:
while use multiline comment in a single line?

Reply:
*why use* not *while use*

Reply:
// doesn't work in css, always use /* */